javascript/react-js/advanced-concepts.md

```diff
@@ -15,24 +15,24 @@ Good Luck!
 
 #### 1. PropTypes
 
-One common discussion about JavaScript is whether it would be better if you could declare types for variables and properties. Many programmers agree that the pattern of declaring types, which you have in many other programming languages is preferable because it allows you to catch errors, such as passing a string to a variable that should be a number. Therefore, React provides the possibility to declare types using PropTypes. And if that's not enough for you, you could also use [TypeScript](https://www.typescriptlang.org/) with React.
+One common discussion about JavaScript is whether it would be better if you could declare types for variables and properties. Many programmers agree that the pattern of declaring types, which you have in many other programming languages is preferable because it allows you to catch errors, such as passing a string to a variable that should be a number. Therefore, React provides the possibility to declare types using [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html). And if that's not enough for you, you could also use [TypeScript](https://www.typescriptlang.org/) with React, here is an [example](https://youtu.be/Z5iWr6Srsj8) of using typescript with React.
```

Comment:
I do not think we want to muddy the water with typescript at this stage. This should be removed entirely IMO

Reply:
It was only left in tact, more as a "To-do" if we ever wanted to expand lessons for any of these concepts down the line.

Reply:
So should i leave it or remove it? 

Or just remove the video reference for typescript?

Reply:
Remove the video reference. Per discussion among the maintainers, the documentation is the most appropriate entry point for this extra topic.

Reply:
done